### PR TITLE
chore: remove ruby 3 and python 3.9

### PR DESCRIPTION
**Pull request by Benaiah**
This came out of the conversation in https://github.com/netlify/build-image/issues/603, where we discussed removing ruby 3 and python 3.9 in order to minimize the additional size of the build image from new software versions.

### CODE REVIEW

File: included_software.md [hunk 1]
```diff
@@ -17,8 +17,7 @@ The specific patch versions included will depend on when the image was last buil
   * 16 (default)
   * Any version that `nvm` can install.
 * Python - `PYTHON_VERSION`, `runtime.txt`, `Pipfile`
-  * 3.9 (default)
```
Comment (ingride) [inline included_software.md:20]: ruby 3.0 should also be ⚡

**Review by ingride — COMMENTED** [review]
looks good otherwise.

File: included_software.md (hunk 1 above)
Comment (Benaiah) [reply included_software.md:20]: great catch, thank you!

**Comment by Benaiah** [comment]
I've fixed the doc issue you mentioned and added a commit updating the default versions used in `run-build.sh` as per https://github.com/netlify/build-image/pull/605.

**Review by ingride — APPROVED** [review]

**Review by ingride — APPROVED** [review]
🐍

**Comment by straight-shoota** [comment]
I don't really follow why remvoing the most recent Python release would be considered a good idea. If you want to save space, I'd recommend throwing out *old* releases.

**Pull request merged**